exercises/concept/remote-control-competition/.docs/instructions.md

```diff
@@ -65,6 +65,6 @@ List<ProductionRemoteControlCar> unsortedCars = new ArrayList<>();
 unsortedCars.add(prc1);
 unsortedCars.add(prc2);
 List<ProductionRemoteControlCar> rankings = TestTrack.getRankedCars(unsortedCars);
-// => rankings.get(0) == prc2
-// => rankings.get(1) == prc1
+// => rankings.get(0) == prc1
+// => rankings.get(1) == prc2
```

Comment:
Nice spot in picking up this error! With this expected order, I wonder if it is clear enough that `getRankedCars` is expected to sort the list (the example now doesn't show a change because the list already happens to be in the correct order). I suggest either changing the order in which the cars are added in the example or swapping the number of victories to match the `rankTwoCars` test instead.

Reply:
I agree!

Reply:
Understood. So to be clear, should I update the changes you suggested in the same commit(by squashing) or add a new commit?

Reply:
A new commit is fine. PRs are automatically squashed when merging so no need to worry about rebasing on your branch!

Reply:
Oh, that's nice. Thank you! I made the change. Could you kindly check?